src/preamble.js

```diff
@@ -1830,7 +1830,7 @@ function integrateWasmJS() {
 
   // Create the wasm instance.
   // Receives the wasm imports, returns the exports.
-  function createWasm(global, env, providedBuffer) {
+  function createWasm(env) {
```

Comment:
oh nice, these were leftovers from the binaryen asm.js support we removed, good to remove them